sunpy/map/map_factory.py

```diff
@@ -4,12 +4,12 @@
 import glob
 from collections import OrderedDict
 import warnings
-
 import numpy as np
 import astropy.io.fits
+from astropy.wcs import WCS
 
 import sunpy
-from sunpy.map.mapbase import GenericMap
+from sunpy.map.mapbase import GenericMap	
```

Comment:
Can you remove the white space at the end of the line?

Reply:
Done